sql/hive/src/test/scala/org/apache/spark/sql/hive/UDFSuite.scala

```diff
@@ -193,4 +193,29 @@ class UDFSuite
       }
     }
   }
+
+  test("SPARK-21318: The correct exception message should be thrown " +
+    "if a UDF/UDAF has already been registered") {
+    val UDAFName = "empty"
+    val UDAFClassName = classOf[org.apache.spark.sql.hive.execution.UDAFEmpty].getCanonicalName
+
+    withTempDatabase { dbName =>
```

Comment:
why do we have to test it inside a database? can't the default database work?

Reply:
@cloud-fan I just copied and modified the code from another test case, the default database works well.

The test case has been simplified now.